--- a/desktop/packages/mullvad-vpn/tasks/distribution.js	2025-07-26 13:38:13.195612653 -0500
+++ b/desktop/packages/mullvad-vpn/tasks/distribution.js	2025-07-26 13:45:47.104522171 -0500
@@ -174,16 +174,6 @@
     },
 
     linux: {
-      target: [
-        {
-          target: 'deb',
-          arch: getLinuxTargetArch(),
-        },
-        {
-          target: 'rpm',
-          arch: getLinuxTargetArch(),
-        },
-      ],
       artifactName: 'MullvadVPN-${version}_${arch}.${ext}',
       category: 'Network',
       icon: distAssets('icon.icns'),
